### DISCUSSION TITLE
Alternatives to ALL CAPs transcription

### DISCUSSION THREAD

**Discussion by CollinBorns** [speechly/speechly#108] [Q&A] [answered]
Are there alternative ways to display the transcribed text vs FULL CAPs?

**Comment by bigdatabaracus** [comment, marked as answer]
At the moment of writing this the API produces only all caps for the transcripts events. No additional capitalisation logic for proper nouns etc is offered. If you wish you can chose to lower case your transcripts and create simple logic for capitalisation upon punctuation. The Speechly [chat demo](https://demos.speechly.com/speech-to-text/) works exactly like this, see implementation [here](https://github.com/speechly/speechly/blob/main/demos/speech-to-text/src/App.tsx#L33).
